src/Relation/Binary/Lattice/Properties/JoinSemilattice.agda

```diff
@@ -101,3 +107,18 @@ dualMeetSemilattice = record
   { _∧_               = _∨_
   ; isMeetSemilattice = dualIsMeetSemilattice
   }
+
+------------------------------------------------------------------------
+-- If ≈ is decidable then so is ≤
+
+≈-dec⇒≤-dec : Decidable _≈_ → Decidable _≤_
+≈-dec⇒≤-dec _≟_ x y with (x ∨ y) ≟ y
+... | yes x∨y≈y = yes (trans (x≤x∨y x y) (reflexive x∨y≈y))
+... | no  x∨y≉y = no (contraposition x≤y⇒x∨y≈y x∨y≉y)
+
+≈-dec⇒dec : Decidable _≈_ → IsDecPartialOrder _≈_ _≤_
```

Comment:
I know it's a bit verbose, but I think the names would be a bit more meaningful if we changed this (and the corresponding ones in the other modules) to `≈-dec⇒isDecPartialOrder`

Reply:
Done.